frontend/js/src/pins/PinRecordingModal.tsx

```diff
@@ -111,7 +117,30 @@ export default NiceModal.create(
       },
       [recordingToPin, blurbContent]
     );
-
+    const updatePinnedRecordingComment = React.useCallback(
+      async (event: React.MouseEvent<HTMLButtonElement>) => {
+        event.preventDefault();
+        try {
+          if (rowId && recordingToPin && currentUser?.auth_token) {
+            await APIService.updatePinRecordingBlurbContent(
+              currentUser.auth_token,
+              rowId,
+              blurbContent
+            );
+            toast.success(
+              <ToastMsg
+                title="Comment updated" message=''/>,
+              {
+                toastId: "pin-update-success",
+              }
+            );
```

Comment:
Here we can have the modal return the updated text so we can update the UI accordingly:
```suggestion
            );
            modal.resolve(blurbContent);
```

I'll leave a comment in PinnedRecordingCard for how to use the returned value.

I don't know if we need to return anything when there is an error, in the catch block below.